.travis.yml

```diff
@@ -13,7 +13,16 @@ install: NODE_ENV=development npm ci
 jobs:
   include:
     - stage: "Tests"
-      name: "Unit Tests"
+      name: "Unit Tests on macOS"
+      os: osx
+      script: npm run test:lint && npm run test:unit && npm run test:danger && npm run test:git-history
+    - name: "Unit Tests on Windows"
+      git:
+        autocrlf: input
+      os: windows
```

Comment:
should we look at using `matrix-expansion` https://docs.travis-ci.com/user/build-matrix/#matrix-expansion to allow for parrarel builds